src/main/java/org/broadinstitute/consent/http/ConsentModule.java

```diff
@@ -200,7 +201,12 @@ OntologyService providesOntologyService() {
 
   @Provides
   OAuthAuthenticator providesOAuthAuthenticator() {
-    return new OAuthAuthenticator(providesSamService());
+    return new OAuthAuthenticator(providesSamService(), providesUserService());
```

Comment:
I was expecting that this code path would no longer need `UserService` now that we have `DuosUser`.

Reply:
Once I remove the user lookup in `OAuthAuthenticator`, I can remove this as well.